pages/linux/xfce4-screenshooter.md

```diff
@@ -0,0 +1,31 @@
+# xfce4-screenshooter
+
+> The XFCE4 screenshot tool.
+
+- Launch the screenshooter GUI:
+
+`xfce4-screenshooter`
+
+- Take a screenshot of the entire screen and launch the GUI to ask how to proceed:
+
+`xfce4-screenshooter --fullscreen`
+
+- Take a screenshot of the entire screen and save it in the specified directory:
+
+`xfce4-screenshooter --fullscreen --save {{path/to/directory}}`
+
+- Wait some time before taking the screenshot:
+
+`xfce4-screenshooter --delay {{seconds}}`
```

Comment:
```suggestion
`xfce4-screenshooter --delay {{s}}`
```

Reply:
I've never seen `{{s}}`. I always saw `{{seconds}}`. Which I think is also what is suggested in the guidelines

> Try to incorporate the spelled-out version of single-letter options in the example's description. The goal is to allow people to understand the syntax of the commands, not just memorize it.

I think it applies to option values too.

Reply:
Yeah, @mebeim - I think we always prefer verbosity here, @mfrw. To this end, I would value `seconds` over `s`.